### ISSUE TITLE
❓ [Question, Support request] Accent changes are not applied after reboot 

### ISSUE THREAD

**Issue by EvantheGrump** [Abdelrhman-AK/WinPaletter#208]
# **❓ Question or support request**

When I set an accent color (Start menu & action center specifically) and reboot, the accent color defaults back to my windows accent color. Is this an intended feature, and can I fix it?

ps. this is my first issue I've ever posted on anything so please excuse any errors I have made. Thanks!

**Comment by Abdelrhman-AK** [comment]
- Please rechange this color (or most colors if possible), press on apply, and then take screenshots to the following registry items in registry editor before and after restarting your Windows, so that I know if the problem affects these colors only or all colors.
               `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Accent`
                `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced`
        `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize`
                     `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\DWM`

- If possible, take screenshots to every items values, not the key only.

**Comment by EvantheGrump** [comment]
Okay, I have done so. Here are the before images: 
![Screenshot 2024-01-02 191417](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/6b5bd3dc-2bdb-49d6-a534-10b2d311b088)
![Screenshot 2024-01-02 191357](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/6e1d92db-7d9f-4b92-a571-30e26aba1aa8)
![Screenshot 2024-01-02 191336](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/5ccba1d7-58b0-4340-9727-7e6cf925b4fa)
![Screenshot 2024-01-02 191312](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/30e92f01-d792-4ef8-89aa-61ab2d22aa28)

**Comment by EvantheGrump** [comment]
and the after images: 
![Screenshot 2024-01-02 191813](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/dd03f1d8-819c-41ee-ab04-2a949668192e)
![Screenshot 2024-01-02 191756](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/4c811134-107c-44cb-927a-4d1042773955)
![Screenshot 2024-01-02 191740](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/4ae4ad8e-3b69-4d6f-bf66-e9f6e36a179d)
![Screenshot 2024-01-02 191717](https://github.com/Abdelrhman-AK/WinPaletter/assets/70341827/6d5a6299-ee74-49c7-b284-34afc40a006c)

**Comment by Abdelrhman-AK** [comment]
I'll investigate about this and try to reproduce this myself, and I I found the bug, I'll fix it.

I'll tell you here what I'll find about this issue.

**Comment by Abdelrhman-AK** [comment]
I tried to reproduce it, but I didn't face this issue.

### Before restart:
![Screenshot (6)](https://github.com/Abdelrhman-AK/WinPaletter/assets/59510211/2ff680a4-2bab-4dbc-9b42-2f33847b433d)

### After restart:
![Screenshot (7)](https://github.com/Abdelrhman-AK/WinPaletter/assets/59510211/59a0724e-8288-4948-a931-2f91473c2bd9)

- Do you have multiple Windows users, or changing user's preferences from another user?
- I have noticed this value, that is not present by default. Try deleting it and tell me if it worked

![Untitled](https://github.com/Abdelrhman-AK/WinPaletter/assets/59510211/7ca74fa0-c3b1-4a49-9320-cf96d2f23058)

![image](https://github.com/Abdelrhman-AK/WinPaletter/assets/59510211/5c24d415-f7b0-47bc-8000-2aa07f4caf1b)

- If not, try setting a new color using Windows settings, not WinPaletter then reuse WinPaletter.

**Comment by EvantheGrump** [comment]
Yes, I have mutliple windows users, but I have not changed the settings for any but the one I use. I have disabled the 'UseNewAutoColorAccentAlgorithm' registry entry, but it has not fixed the problem. When I set the color using windows and then WinPalleter after, the problem gets worse, not applying any color I set.

**Comment by Abdelrhman-AK** [comment]
- I have tested `UseNewAutoColorAccentAlgorithm`. It is not relevant to this issue.
- I was going to tell you to screenshot all value of `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Accent` : `AccentPalette` before and after restart. But this won't be relevant as the issue got worse and not applying any color.
- Can you test older versions of WinPaletter, starting from `1.0.8.3` and lower so I know the changes I made to the way WinPaletter write to registry to handle multiple users are the cause of this issue or not.
